RELEASE-NOTES.md

```diff
@@ -1,5 +1,96 @@
+- Remove all infra overrides-related flags — https://github.com/PrefectHQ/prefect/pull/12742
+
+### Experimental
+
+#### Pydantic V2 Compatibility
+- Refactor `field_validator` and `model_validator` to map Pydantic kwargs between versions — https://github.com/PrefectHQ/prefect/pull/12676
+- Fix type-hinting for self-validating fields — https://github.com/PrefectHQ/prefect/pull/12710
+ 
+#### Events and Automations
+- Implement the `run-deployment` automation action — https://github.com/PrefectHQ/prefect/pull/12677
+- Implement the `send-notification` action — https://github.com/PrefectHQ/prefect/pull/12693
+- Make `TriggeredAction.firing` required — https://github.com/PrefectHQ/prefect/pull/12697
+- Add an Actions service — https://github.com/PrefectHQ/prefect/pull/12699
+- Implement the `call-webhook` action and adds all Action client-side schemata — https://github.com/PrefectHQ/prefect/pull/12728
+- Implement `change-flow-run-state`, `cancel-flow-run`, and `suspend-flow-run` — https://github.com/PrefectHQ/prefect/pull/12730
+- Add functions for querying and counting events — https://github.com/PrefectHQ/prefect/pull/12696
+- Implement the `pause-deployment` and `resume-deployment` actions — https://github.com/PrefectHQ/prefect/pull/12733
+- Add `/events/filter` and `/events/count-by` route trees — https://github.com/PrefectHQ/prefect/pull/12736
+- Allow for creating automations via deployments when experimental events is on — https://github.com/PrefectHQ/prefect/pull/12701
+- Add ability to stream out events via websocket — https://github.com/PrefectHQ/prefect/pull/12744
+- Implementing the `pause-automation` and `resume-automation` actions — https://github.com/PrefectHQ/prefect/pull/12738
+- Add an Automations CLI — https://github.com/PrefectHQ/prefect/pull/12754
+- Rename `prefect-cloud.*` events and labels to `prefect.*` — https://github.com/PrefectHQ/prefect/pull/12755
+- Add ability to emit events to an ephemeral Prefect server — https://github.com/PrefectHQ/prefect/pull/12762
+- Disable `events` and `automations` API routes when experimental events setting is not enabled — https://github.com/PrefectHQ/prefect/pull/12777
+- Add compatibility tests for client and server triggers and actions — https://github.com/PrefectHQ/prefect/pull/12778
+- Disable the automations integration flows for Prefect Cloud — https://github.com/PrefectHQ/prefect/pull/12784
+- Implement the work pool and work queue actions — https://github.com/PrefectHQ/prefect/pull/12735
+- Add helper functions for creating an events client or subscriber — https://github.com/PrefectHQ/prefect/pull/12759
+
+### Documentation
+- Update example to `prefect.yaml` in  work-pools concepts page — https://github.com/PrefectHQ/prefect/pull/12695
+- Fix stray link — https://github.com/PrefectHQ/prefect/pull/12721
+- Add hyphen — https://github.com/PrefectHQ/prefect/pull/12724
+- Fix typo — https://github.com/PrefectHQ/prefect/pull/12722
```

Comment:
these may not be worth including, esp if we don't feel like being descriptive

Reply:
+1